scripts/import_osm.py

```diff
@@ -0,0 +1,111 @@
+from sqlalchemy import create_engine
+from sqlalchemy.engine.base import Engine
+from sqlalchemy.schema import CreateSchema
+from geoalchemy2 import Geometry, WKTElement
+from typing import Dict
+
+# test simple import now, convert to module later
+sys.path.insert(0, "..")
+from models import OSMPoint
+from osm_tags import tags_to_filter
+
+
+class OsmImporter(object):
+    """Import OSM data to PostGIS using Overpass API."""
+
+    def __init__(self, args: Dict):
+        args = self._parse_args(args)
+        self.slug = args["slug"]
+        self.bbox = args["bbox"]
+        self.city = args["city_name"]
+        self._engine = self._create_engine()
+
+    @staticmethod
+    def _parse_args(args) -> Dict:
+        """Parse input arguments to a dict."""
+        slug = args["city"]
+        if not slug:
+            raise AssertionError("You must specify the city name.")
+        # BBOX (minx, miny, maxx, maxy) takes precedence over city
+        arg_bbox = args["bbox"]
+        if arg_bbox:
+            minx, miny, maxx, maxy = map(float, arg_bbox.split(", "))
+            city_name = ""
+        else:
+            # support old city parameter too, if we want to let osmnx do geocoding
+            if slug == "COP":
+                city_name = "Copenhagen, Denmark"
+                minx, miny, maxx, maxy = 12.42000, 55.61000, 12.65000, 55.78000
+            elif slug == "HEL":
+                city_name = "Helsinki, Finland"
+                minx, miny, maxx, maxy = 24.82345, 60.14084, 25.06404, 60.29496
+            elif slug == "WAR":
+                city_name = "Warsaw, Poland"
+                minx, miny, maxx, maxy = 20.79057, 52.09901, 21.31300, 52.38502
+            else:
+                raise AssertionError("You must specify a bounding box or a known city.")
+
+        return {"slug": slug, "bbox": (minx, miny, maxx, maxy), "city_name": city_name}
+
+    def _create_engine(self) -> Engine:
+        """Sets up and returns a DB engine."""
+        sql_url = get_connection_url(dbname="geoviz")
+        engine = create_engine(sql_url)
+        return engine.execution_options(
+            schema_translate_map={"schema": self.slug}
+        )
+
+    def run(self):
+        self._initialise_db()
+        print("Fetching OSM data from overpass API...")
+        pois = self._get_amenities()
+        pois = pois.to_crs(epsg=3035)
+        pois.geometry = pois.centroid
+        pois = pois.to_crs(epsg=4326)
+
+        pois = pois.reset_index()
+        tag_columns = list(pois.columns)
+        tag_columns.remove("osmid")
+        tag_columns.remove("geometry")
+
+        pois = pois.rename(columns={"osmid": "node_id"})
+        pois = pois.rename(columns={"geometry": "geom"})
+        pois = pois.set_index("node_id")
+        pois["geom"] = pois["geom"].apply(lambda geom: WKTElement(geom.wkt, srid=4326))
+
+        pois["tags"] = pois[tag_columns].apply(lambda x: x.to_json(), axis=1)
+        pois = pois.drop(tag_columns, axis=1)
+
```

Comment:
I think we should add here some basic logging so the user running the script sees if anything has been imported. Just some lines like in the other importers like
`self.logger.info(f"Found {len(walk_access_dict)} accessibility nodes, importing...")` and
`self.logger.info(f"Saving {len(nodes_to_save)} accessibility nodes...")`